### PR TITLE
refuse to perform mapping operations in /.baked/

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by birarda** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2898/HighFidelity-Beta-PR11270-4414b5bf618915e395155d0154858f3012876c9f.exe

File: libraries/baking/src/TextureBaker.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ const QStringList TextureBaker::getSupportedFormats() {
     auto formats = QImageReader::supportedImageFormats();
     QStringList stringFormats;
     std::transform(formats.begin(), formats.end(), std::back_inserter(stringFormats),
-                   [](auto& format) -> QString { return format; });
+                   [](QByteArray& format) -> QString { return format; });
```
Comment (Atlante45) [inline libraries/baking/src/TextureBaker.cpp]: I don't really understand why this works. QByteArray, doesn't have implicit QString conversion.
I'd do that instead:
```cpp
    std::transform(formats.begin(), formats.end(), std::back_inserter(stringFormats),
                   [](QByteArray& format) { return QString::fromUtf8(format); });
```

Comment (birarda) [reply libraries/baking/src/TextureBaker.cpp]: This was @huffman's code other than the change from auto but it does have a ctor that takes QByteArray and uses fromUtf8.

http://doc.qt.io/qt-5/qstring.html#QString-8

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2922/HighFidelity-Beta-PR11270-3abca67f1a48426162e3c56e5e29743d4bea0522.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2924/HighFidelity-Beta-PR11270-ed21ad11d27f60eacb8f94b34addca4031259ef5.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2933/HighFidelity-Beta-PR11270-f2d67b91f7f31cd53353dbcc663b88ab363ec9df.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2941/HighFidelity-Beta-PR11270-84b5afaa4dd7244efb93a423ff8559e25977bd3c.exe

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2946/HighFidelity-Beta-PR11270-84b5afaa4dd7244efb93a423ff8559e25977bd3c.exe

File: libraries/networking/src/AssetUtils.cpp [hunk 2]
```diff
@@ -99,5 +97,8 @@ QString bakingStatusToString(BakingStatus status) {
             return "Baked";
         case Error:
             return "Error";
+        default:
+        case Unrelevant:
+            return "--";
```
Comment (huffman) [inline libraries/networking/src/AssetUtils.cpp]: More a personal thing, but can we put default at the bottom?

File: libraries/baking/src/TextureBaker.cpp (hunk 1 above)
Comment (huffman) [reply libraries/baking/src/TextureBaker.cpp]: Did the auto not work on other platforms?

File: libraries/networking/src/AssetUtils.cpp (hunk 2 above)
Comment (Atlante45) [reply libraries/networking/src/AssetUtils.cpp]: The case is unnecessary even. default will catch it.

File: libraries/baking/src/TextureBaker.cpp (hunk 1 above)
Comment (Atlante45) [reply libraries/baking/src/TextureBaker.cpp]: That's a cpp14 feature.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2969/HighFidelity-Beta-PR11270-da8ae237565abe0bf21496ef25bdff09d2b5ce90.exe

**Pull request merged**